_data/changelogs/docs-settings.yml

```diff
@@ -2,6 +2,19 @@ title: Settings
 type: utility
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Updated sass map instructions to reflect new pattern.
+    summaryAdditional: Users are now instructed to define only the variable they wish to change instead of having to define the entire map
+    isBreaking: true
```

Comment:
Updated the `Breaking Change` section with the following:

> Previously, if a user left an item out of a map, it would be set to `false` and thus deactivating it. After this update, an item left out of the map will maintain its default value.
>
> If users deactivated certain settings by not declaring them, this update will cause those variables to be activated. Note that we did not recommend that pattern, but it's possible for users to have done so on their own